src/Game/Scripting/Aliases.cs

```diff
@@ -1,15 +1,107 @@
-﻿namespace ClassicUO.Game.Scripting
+﻿using System;
+using System.Collections.Generic;
+using static ClassicUO.Game.Scripting.Interpreter;
+
+namespace ClassicUO.Game.Scripting
 {
     public static class Aliases
     {
+        // Called by upper management class to register all the desired aliases
         public static void Register()
         {
```

Comment:
As you mentioned about Razor, that was indeed the way Aliases worked in the original source code before my changes. So, if the script had the word "last", "mount" or "backpack" for a UINT type (such as Serial) it would convert it to the expected Serial and all was working fine.

Nevertheless, when I started working with commands accepting Graphic or Color (or even Source), I noticed a problem. For instance, both Graphic and Color are USHORT types, but the word "any" can be converted to a MaxValue for Color, but it cannot be converted for Graphic (as it is invalid).

That is when I noticed Global Aliases (without any scope/context) were having some limitations, for example, with the commands:
- useonce (graphic) [color] -> where "any" can only be converted as an alias for the Color param.
- movetype (graphic) (source) -> where alias such as "any" and "ground" can be used (unrelated to Serial"

So I added another type of Alise, one that is restricted to a certain scope (in this case a specific parameter, such as Color, Serial, Source, Graphic, etc).

Not sure if that was the best choice, but that was the reasoning behind my change :)

Reply:
Aliases are string names for particular serials, resolved at run time. For example, 'mount' and 'backpack' resolve to serials. But the graphic and color parameters in your example do not resolve to serials. They instead have a specific meaning for that command and parameter alone. Typically, each command will handle exactly what 'any' means in it's own handler.

Reply:
But @jaedan, any specific reason why Aliases are only for Serial? Is it more of a technical need or your preference?

Like, with your generic implementation of the AST, doesn't it make sense to have a more generic concept for Aliases as well? So far I have not found a command doing custom handling of aliases. Meaning the Color "any" means the same for all commands, right?

If my argument did not convince you and you prefer Aliases being only for Serial, I am ok with the decision and I can change the class back. Then I could just add this making of other types in the ParameterList "default" handling o avoid having each command reimplementing such check. Would you prefer something like this?